stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/request/AnyChannelPaginationRequest.kt

```diff
@@ -24,21 +24,11 @@ internal fun AnyChannelPaginationRequest.hasFilter(): Boolean {
 }
 
 internal fun AnyChannelPaginationRequest.isFirstPage(): Boolean {
-    return messageFilterDirection == null
+    return channelOffset == 0
```

Comment:
We use this extension property in `queryChannels` only flow. But there we never set `messageFilterDirection` that's why it worked incorrectly